generated/proto/encodingpb/types.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option go_package = "github.com/xichen2020/eventdb/generated/proto/encodingpb";
+
+package encodingpb;
+
+enum CompressionType {
+    UNKNOWN_COMPRESSION = 0;
+    ZSTD = 1;
+}
+
+enum EncodingType {
+    UNKNOWN_ENCODING = 0;
+    LENGTH = 1;
+    DICTIONARY = 2;
+    TABLE = 3;
+    DELTA = 4;
```

Comment:
Perhaps add the others when we have implementations for those?

Reply:
👍